src/dialects/abstract/query-generator.types.ts

```diff
@@ -0,0 +1,4 @@
+export interface QueryGeneratorDropSchemaQueryObject {
+  query: string;
+  bind?: unknown[];
```

Comment:
BindOrReplacements is the correct type for this

```suggestion
  bind: BindOrReplacements;
```

Also, if someone uses `QueryWithBindParams` instead of just a string, I'd expect this property to be mandatory

Reply:
Issue is that this is actually returned via `this.queryGenerator.dropSchemaQuery`, which doesn't need the `bind` param. How would we like to handle that case?

Reply:
The typing of `dropSchemaQuery` is incorrect, when `dropSchemaQuery` returns an object, `bind` is always present

Reply:
Gotcha - fixed that method's typing as well.